index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@700&display=swap" rel="stylesheet">
+    <title>My Portfolio</title>
+</head>
+<body>
+    
+    <!-- navbar section -->
+<nav class="navbar">
+    <h3 id="logo">Dela McRaph</h3>
+    <ul>
+        <li><a href="#">Portfolio</a></li>
+        <li><a href="#">About</a></li>
+        <li><a href="#">Contact</a></li>
+    </ul>
+
+</nav>
+
+<!-- self introduction -->
+<div class="container">
+     <div class="intro"> 
+        <h4 class="primary-text">
+            I AM RAPHAEL. <br>
+            GLAD TO SEE YOU!
+        </h4>
+        <P class="primary-text-details">
+            I'm a software developer! I can help you build a product, feature or website look through some of my work and experience! 
+            if you like what you see and have a project you need coded, don't hesitate 
+            to contact me.
+        </P>
+        <p class="lets-connect">
+            Lets Connect
+        </p>

```

Comment:
- [ ] Kindly improve your **UI**, your design is overflowing and looks very bad. In the web development world so many times you will hear **BAD USER EXPERIENCE** or **GOOD UX**. Try to make your design looks similar to `Figma` design as much as possible. You can take a look at your design and required design as given below:

### Your Design
![Screenshot from 2023-04-27 19-54-05](https://user-images.githubusercontent.com/66476329/234892319-56945a06-2d3f-4c71-9f12-e7b53bdc00f4.png)

### Design should look like

![Screenshot from 2023-04-27 19-55-11](https://user-images.githubusercontent.com/66476329/234892607-d8af9744-54ca-4a69-9477-be8c8369ab21.png)